src/app/examples/modal-example/modal-example.component.ts

```diff
@@ -14,7 +14,7 @@ export class ModalExampleComponent {
   showModal() {
     const config: ModalConfig = {
       title: 'First Embedded Modal',
-      titleHorizontalAlignment: 'center',
+      closeBtnPlacement: 'inside',
```

Comment:
Consider renaming to:
```suggestion
      closeBtnPosition: 'inside',
```